python/taichi/lang/kernel_impl.py

```diff
@@ -618,21 +658,30 @@ def func__(*args):
                         ndarray_type.NdarrayType) and (self.match_ext_arr(v)):
                     has_external_arrays = True
                     is_numpy = isinstance(v, np.ndarray)
+                    is_torch = isinstance(v, torch.Tensor)
```

Comment:
btw if you only installed Paddle, this will fail with "unrecognized module 'torch'". Possibly replace with: `is_torch = isinstance(v, torch.Tensor) if has_torch else False`

Reply:
Thanks! I'll fix the problem.